src/sentry/api/serializers/models/dashboard.py

```diff
@@ -166,8 +188,28 @@ def serialize(self, obj, attrs, user, **kwargs):
         return data
 
 
+class DashboardFilters(TypedDict):
+    release: list[str]
+
+
+class DashboardDetailsResponse(TypedDict):
+    id: str
+    title: str
+    dateCreated: str
+    createdBy: list[OpaqueSerializedResponse]
```

Comment:
@sentaur-athena I wasn't sure how to type this `createdBy` field. It's derived from [this line of code](https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/dashboard.py#L197) which makes a request to the user service and it lists `list[OpaqueSerializedResponse]` as the return of the `serialize_many` call.

This is supposed to be a user object. Is there an example of a place where this type of data is typed properly?